src/styles/utils/variables.scss

```
@@ -0,0 +1,20 @@
+$one-day-width: 100px;
+$one-day-margin: 1px;
+$one-day-indent: $one-day-width + 2 * $one-day-margin;
+$grey-color: #eee;
+$calendar-width: ($one-day-width + 2 * $one-day-margin) * 7;
```

```suggestion
$calendar-width: ($one-day-width + (2 * $one-day-margin) + 1) * 7;
```
Offset `margin-right: 1px;` [image](https://user-images.githubusercontent.com/57752461/133618762-b3a093d5-7171-4e86-a6df-e16e9460ad08.png)